### DISCUSSION TITLE
Remove old deployments

### DISCUSSION THREAD

**Discussion by WayneWayner** [subquery/subql#933] [Frontier TestNet - Bug Reporting]
Is it possible to add a feature to remove old deployments which are not in your local database anymore? There is no possibility to do it from the 8000 dashboard nor from the explorer.

8000 dashboard:
![subquery_8000_dash](https://user-images.githubusercontent.com/60453944/165628585-b8f2f1fa-13c4-40c5-990b-9dc1c3fbc897.png)

Create a plan:
![subquery_deployments](https://user-images.githubusercontent.com/60453944/165627829-6067a7fc-1239-4ab6-9252-3d6c1a39ac67.png)

My profile (explorer): 
![grafik](https://user-images.githubusercontent.com/60453944/165630307-86690ae9-ca98-46ce-a5c5-f9fbdb7f0feb.png)

In my opinion it would be nice to have the stop indexing command at your indexer profile. Or even all of these command types.

**Comment by jamesbayly** [comment]
We should also add this to the hosted service @zhex

**Reply by zhex** [reply]
in the hosted service, the user controls the deployments directly (create/update/delete).  so it should be 100% match, with no similar issue.

**Comment by ianhe8x** [comment]
@CaiYiLiang We can 
* add some filter in this dropdown to filter out stopped deployments. 
* add version along with the project name so it is easier for people to tell which deployment it is.

**Comment by lilian79511** [comment]
Ticket is created: SQN-412
